<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Rogalicious" failures="0" tests="6" name="com.revature.bugcatcher.DefectRunner" time="5.515" errors="0" timestamp="2022-10-26T19:12:55 PDT" skipped="0">
  <testcase classname="com.revature.bugcatcher.DefectRunner" name="runScenario" time="0.731"/>
  <system-out/>
  <testcase classname="com.revature.bugcatcher.DefectRunner" name="runScenario" time="0.813"/>
  <system-out/>
  <testcase classname="com.revature.bugcatcher.DefectRunner" name="runScenario" time="0.728"/>
  <system-out/>
  <testcase classname="com.revature.bugcatcher.DefectRunner" name="runScenario" time="0.740"/>
  <system-out/>
  <testcase classname="com.revature.bugcatcher.DefectRunner" name="runScenario" time="1.791"/>
  <system-out/>
  <testcase classname="com.revature.bugcatcher.DefectRunner" name="runScenario" time="0.712"/>
  <system-out/>
</testsuite> <!-- com.revature.bugcatcher.DefectRunner -->
